src/Service/Fido2Service.cs

```diff
@@ -292,8 +294,8 @@ public async Task<string> CreateSigninToken(SigninTokenRequest request)
         {
             Success = true,
             UserId = request.UserId,
-            Timestamp = DateTime.UtcNow,
-            ExpiresAt = DateTime.UtcNow.AddSeconds(120),
+            Timestamp = _timeProvider.GetUtcNow().DateTime,
+            ExpiresAt = _timeProvider.GetUtcNow().DateTime.Add(request.TimeToLiveTimeSpan),
```

Comment:
What's the default value?

Reply:
default value is set in the SigninTokenRequest model. It is 120 seconds currently